randovania/games/prime2/logic_database/Sanctuary Fortress.txt

```diff
@@ -1001,7 +1001,7 @@ Dynamo Works
 Extra - asset_id: 3222355176
 Extra - in_dark_aether: False
 Extra - low_memory_mode: True
-Hint Features - Bomb Slot, Boost Pipe, Luminoth Corpse, Spider Track
+Hint Features - Bomb Slot, Boost Pipe, Keybearer Corpse, Luminoth Corpse, Morph Ball Tunnel, Spider Track
```

Comment:
strictly speaking the spider guardian item isn't inside the tunnel!

Reply:
dangit I knew I overlooked something. Will fix